src/screens/SendCoinsScreen.tsx

```diff
@@ -482,13 +484,15 @@ class SendCoinsScreen extends Component<Props, State> {
         <Text style={styles.radioButtonsTitle}>{i18n.send.transaction.type}</Text>
 
         <RadioButton
+          testID="send-coins-secure-transaction-type-radio-button"
```

Comment:
In other places, I named testID for radio buttons without the button word at the end. So, according to my convention it'd be: `testID="send-coins-secure-transaction-type-radio"`. 

FYI only, you don't need to change it if you like how you did it more.

Reply:
I have no very strong opinion on this matter. I will change that to your convention.